fiboa_cli/datasets/pt.py

```diff
@@ -0,0 +1,84 @@
+from ..convert_utils import convert as convert_
+import geopandas as gpd
+import pandas as pd
+import fiona
+
+SOURCES = "https://www.ifap.pt/isip/ows/resources/2023/Continente.gpkg"
+#SOURCES = "https://www.ifap.pt/isip/ows/resources/2022/2022.zip"
+
+ID = "pt"
+TITLE = "Field boundaries for Portugal (identificação de parcelas)"
+DESCRIPTION = """Open field boundaries from Portugal"""
+PROVIDER_NAME = "IPAP - Instituto de Financiamento da Agricultura e Pescas"
+PROVIDER_URL = "https://www.ifap.pt/isip/ows/"
+ATTRIBUTION = None
+
+# Inspire license. Not 100% clear at source
+LICENSE = {"title": "No conditions apply", "href": "https://inspire.ec.europa.eu/metadata-codelist/ConditionsApplyingToAccessAndUse/noConditionsApply", "type": "text/html", "rel": "license"}
+
+COLUMNS = {
+    "geometry": "geometry",
+    "OSA_ID": "id",
+    "CUL_ID": "block_id",
+    "CUL_CODIGO": "crop_code",
+    "CT_português": "crop_name",
+    "Shape_Area": "area"
```

Comment:
Can we make use of the Shape_Length as well? For perimeter...

Reply:
Thanks. I was unaware of the usefulness, added.